sdk/dotnet/Ec2/CustomerGateway.cs

```diff
@@ -49,7 +49,7 @@ public partial class CustomerGateway : Pulumi.CustomResource
         /// The gateway's Border Gateway Protocol (BGP) Autonomous System Number (ASN).
         /// </summary>
         [Output("bgpAsn")]
-        public Output<int> BgpAsn { get; private set; } = null!;
```

Comment:
So, do we have our breaking changes policy in place yet? We need to apply it...

Reply:
This is because of string to int right? This is actually a change here https://github.com/terraform-providers/terraform-provider-aws/pull/14030

It seems like TF was actually making this a strong incorrectly :/

I think we are going to have to accept this with a changelog entry as the api upstream (aws) requires an int

Reply:
Ah, let's add an explicit changelog and ship it then

Reply:
I will add that at merge time to save waiting on another build for CHANGELOG